packages/commonwealth/client/scripts/hooks/useAnimation.ts

```diff
@@ -0,0 +1,25 @@
+import { useEffect, useState } from 'react';
+
+export const useAnimation = () => {
+  const [animationStyles, setAnimationStyles] = useState({
+    opacity: 0,
+    transform: 'translateY(40px)',
+    transition: '1s',
+  });
+
+  useEffect(() => {
+    const timeoutId = setTimeout(() => {
+      setAnimationStyles({
+        opacity: 1,
+        transform: 'translateY(0)',
+        transition: '1s',
+      });
+    }, 100);
+
+    return () => {
+      clearTimeout(timeoutId);
+    };
+  }, []);
```

Comment:
We have a `useNecessaryEffect` hook that deals with timeouts in a similar way, I suggest we use that here instead

Reply:
thanks for heads up, changed